### ISSUE TITLE
Windows library doesn't seems to exist.

### ISSUE THREAD

**Issue by JakeCB** [esensar/kotlinx-serialization-msgpack#60]
**Version**
 - Library version: 0.4.2
 - Platform info: Windows

**Describe the bug**
After adding the dependency to commonMain the build seems to think there's not a compatible version available.

```       
val commonMain by getting {
            dependencies {
                implementation(kotlin("stdlib-common"))
                implementation("io.ktor:ktor-client-core:$ktorVersion")
                implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:$serializationVersion")
                implementation("org.jetbrains.kotlinx:kotlinx-serialization-protobuf:$serializationVersion")
                implementation("com.ensarsarajcic.kotlinx:serialization-msgpack:${messagePackVersion}")
            }
        }
```

```
Execution failed for task ':compileKotlinNative'.
> Could not resolve all files for configuration ':nativeCompileKlibraries'.
   > Could not resolve com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2.
     Required by:
         project :
      > No matching variant of com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 was found. The consumer was configured to find a usage of 'kotlin-api' of a library, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native', attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64' but:
          - Variant 'commonMainMetadataElements' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a usage of 'kotlin-api' of a component:
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'common' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native'
              - Other compatible attributes:
                  - Doesn't say anything about its component category (required a library)
                  - Doesn't say anything about org.jetbrains.kotlin.native.target (required 'mingw_x64')
          - Variant 'iosArm64ApiElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'ios_arm64' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'iosArm64MetadataElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a usage of 'kotlin-metadata' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'ios_arm64' and the consumer needed a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'iosX64ApiElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'ios_x64' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'iosX64MetadataElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a usage of 'kotlin-metadata' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'ios_x64' and the consumer needed a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'jsIrApiElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a usage of 'kotlin-api' of a component:
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'js' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native'
              - Other compatible attributes:
                  - Doesn't say anything about its component category (required a library)
                  - Doesn't say anything about org.jetbrains.kotlin.native.target (required 'mingw_x64')
          - Variant 'jsIrRuntimeElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2:
              - Incompatible because this component declares a usage of 'kotlin-runtime' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'js' and the consumer needed a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native'
              - Other compatible attributes:
                  - Doesn't say anything about its component category (required a library)
                  - Doesn't say anything about org.jetbrains.kotlin.native.target (required 'mingw_x64')
          - Variant 'jsLegacyApiElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a usage of 'kotlin-api' of a component:
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'js' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native'
              - Other compatible attributes:
                  - Doesn't say anything about its component category (required a library)
                  - Doesn't say anything about org.jetbrains.kotlin.native.target (required 'mingw_x64')
          - Variant 'jsLegacyRuntimeElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2:
              - Incompatible because this component declares a usage of 'kotlin-runtime' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'js' and the consumer needed a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native'
              - Other compatible attributes:
                  - Doesn't say anything about its component category (required a library)
                  - Doesn't say anything about org.jetbrains.kotlin.native.target (required 'mingw_x64')
          - Variant 'jvmApiElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares an API of a component:
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native'
              - Other compatible attributes:
                  - Doesn't say anything about its component category (required a library)
                  - Doesn't say anything about org.jetbrains.kotlin.native.target (required 'mingw_x64')
          - Variant 'jvmRuntimeElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a runtime of a component:
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native'
              - Other compatible attributes:
                  - Doesn't say anything about its component category (required a library)
                  - Doesn't say anything about org.jetbrains.kotlin.native.target (required 'mingw_x64')
          - Variant 'metadataApiElements' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2:
              - Incompatible because this component declares a usage of 'kotlin-metadata' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'common' and the consumer needed a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native'
              - Other compatible attributes:
                  - Doesn't say anything about its component category (required a library)
                  - Doesn't say anything about org.jetbrains.kotlin.native.target (required 'mingw_x64')
          - Variant 'nativeApiElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'macos_x64' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'nativeMetadataElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a usage of 'kotlin-metadata' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'macos_x64' and the consumer needed a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'tvosArm64ApiElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'tvos_arm64' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'tvosArm64MetadataElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a usage of 'kotlin-metadata' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'tvos_arm64' and the consumer needed a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'tvosX64ApiElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'tvos_x64' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'tvosX64MetadataElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a usage of 'kotlin-metadata' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'tvos_x64' and the consumer needed a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'watchosArm32ApiElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'watchos_arm32' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'watchosArm32MetadataElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a usage of 'kotlin-metadata' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'watchos_arm32' and the consumer needed a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'watchosArm64ApiElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'watchos_arm64' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'watchosArm64MetadataElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a usage of 'kotlin-metadata' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'watchos_arm64' and the consumer needed a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'watchosX64ApiElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'watchos_x64' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)
          - Variant 'watchosX64MetadataElements-published' capability com.ensarsarajcic.kotlinx:serialization-msgpack:0.4.2 declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'native':
              - Incompatible because this component declares a usage of 'kotlin-metadata' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'watchos_x64' and the consumer needed a usage of 'kotlin-api' of a component, as well as attribute 'org.jetbrains.kotlin.native.target' with value 'mingw_x64'
              - Other compatible attribute:
                  - Doesn't say anything about its component category (required a library)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

```

**Comment by esensar** [comment]
Oh, sorry about that. I had an issue with building Windows at one point and forgot to re-enable it.
Thank you for reporting this, I will get it fixed.

**Comment by esensar** [comment]
This will be fixed in 0.4.3, which will be released soon. Thank you for reporting!
